src/supremm/datasource/prometheus/promsummarize.py

```diff
@@ -0,0 +1,301 @@
+import os
+import time
+import logging
+import datetime
+
+import requests
+import numpy as np
+
+from supremm.datasource.prometheus.prominterface import PromClient, Context
+from supremm.plugin import NodeMetadata
+from supremm.summarize import Summarize
+
+
+VERSION = "2.0.0"
```

Comment:
Don't change the version

Reply:
Resolved in:
https://github.com/ubccr/supremm/pull/259/commits/a34241f9050f4e6fcc5b9fb084199663ce3c9fe2
https://github.com/ubccr/supremm/pull/259/commits/e928967e850ddb22068e1e23ae33fa39e58e0b23
https://github.com/ubccr/supremm/pull/259/commits/c7b17840d6afddd9945104bdc48378a428543fb2